src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -664,14 +664,14 @@ An event that fires at the end of page navigation, after the new page is visible
 
 The `<ViewTransitions />` component fires this event both on initial page navigation for a pre-rendered page and on any subsequent navigation, either forwards or backwards.
 
-You can use this event to run code on every page navigation, or only once ever:
+You can use this event to run code on every page navigation. That is especially useful when it comes to setting up event listeners that would otherwise be lost during navigation.
```

Comment:
I think we can turn the second sentence into an example:

```suggestion
You can use this event to run code on every page navigation, for example to set up event listeners that would otherwise be lost during navigation.
```